src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -175,50 +176,90 @@ const serverHostZeroDeprecation: ConfigDeprecation = (settings, fromPath, addDep
   return settings;
 };
 
+const removeFromConfigStep = (setting: string) => {
+  return i18n.translate('core.deprecations.common.removeFromConfig', {
+    defaultMessage: `Remove "{setting}" from your kibana configs.`,
```

Comment:
> defaultMessage: `Remove "{setting}" from your kibana configs.`

Do folks often use more than one config? Maybe we should change that to plain `configuration`?

Reply:
I reused the label from an existing deprecation, but you're right, `kibana configuration` seems better than `kibana configs`. I doubt even 0.1% of our customers uses multiple config files